Sources/Valkey/Connection/ValkeyConnection.swift

```diff
@@ -56,6 +67,17 @@ public final actor ValkeyConnection: ValkeyClientProtocol, Sendable {
         self.configuration = configuration
         self.id = connectionID
         self.logger = logger
+        #if DistributedTracingSupport
+        self.tracer = configuration.tracing.tracer
+        #endif
+        switch address?.value {
```

Comment:
Given `address` is now declared inside a `#if DistributedTracingSupport` its initialisation will also need to be conditional compiled.